<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">alternate-background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="b_equals">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>210</y>
      <width>30</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">CalcPushButtons</string>
    </attribute>
   </widget>
   <widget class="QLineEdit" name="e_input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="t_output">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>35</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="l_Rectangular">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rectangular:</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_Polar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Polar:      a =</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_degrees">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Degrees</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">CalcRadioButtons</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radio_radians">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Radians</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">CalcRadioButtons</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radio_grads">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>80</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Grads</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">CalcRadioButtons</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>31</width>
      <height>53</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="l_x">
       <property name="text">
        <string>x =</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_y">
       <property name="text">
        <string>y =</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_z">
       <property name="text">
        <string>z =</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="e_x">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_y">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_z">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="b_plus">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>250</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">CalcPushButtons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="b_minus">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>210</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">CalcPushButtons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="b_divide">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>130</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">CalcPushButtons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="b_multiply">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>170</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">CalcPushButtons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="b_8">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">CalcPushButtons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="b_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>210</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">CalcPushButtons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="b_point">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>250</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">CalcPushButtons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="b_5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>170</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">CalcPushButtons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="b_1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">CalcPushButtons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="b_0">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>250</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">CalcPushButtons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="b_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>170</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">CalcPushButtons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="b_6">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>170</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">CalcPushButtons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="b_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">CalcPushButtons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="b_9">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>130</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">CalcPushButtons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="b_7">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">CalcPushButtons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="b_sin">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">CalcPushButtons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="b_cos">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">CalcPushButtons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="b_tan">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">CalcPushButtons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="b_mod">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>250</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">CalcPushButtons</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="l_asterisk">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_angle">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>170</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>/_</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">CalcPushButtons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="b_conj">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>250</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>conj</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">CalcPushButtons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="b_arg">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>arg</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">CalcPushButtons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="b_mag">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>210</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>mag</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">CalcPushButtons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="b_clear">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">CalcPushButtons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="b_parenth_r">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>)</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">CalcPushButtons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="b_parenth_l">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>(</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">CalcPushButtons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="b_j">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>130</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>j</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">CalcPushButtons</string>
    </attribute>
   </widget>
   <zorder>l_asterisk</zorder>
   <zorder>b_equals</zorder>
   <zorder>t_output</zorder>
   <zorder>l_Rectangular</zorder>
   <zorder>l_Polar</zorder>
   <zorder>radio_degrees</zorder>
   <zorder>radio_radians</zorder>
   <zorder>radio_grads</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>b_plus</zorder>
   <zorder>b_minus</zorder>
   <zorder>b_divide</zorder>
   <zorder>b_multiply</zorder>
   <zorder>b_8</zorder>
   <zorder>b_2</zorder>
   <zorder>b_point</zorder>
   <zorder>b_5</zorder>
   <zorder>b_1</zorder>
   <zorder>b_0</zorder>
   <zorder>b_4</zorder>
   <zorder>b_6</zorder>
   <zorder>b_3</zorder>
   <zorder>b_9</zorder>
   <zorder>b_7</zorder>
   <zorder>b_sin</zorder>
   <zorder>b_cos</zorder>
   <zorder>b_tan</zorder>
   <zorder>b_mod</zorder>
   <zorder>e_input</zorder>
   <zorder>e_x</zorder>
   <zorder>e_y</zorder>
   <zorder>e_z</zorder>
   <zorder>b_angle</zorder>
   <zorder>b_conj</zorder>
   <zorder>b_arg</zorder>
   <zorder>b_mag</zorder>
   <zorder>b_clear</zorder>
   <zorder>b_parenth_r</zorder>
   <zorder>b_parenth_l</zorder>
   <zorder>b_j</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="CalcRadioButtons"/>
  <buttongroup name="CalcPushButtons"/>
 </buttongroups>
</ui>
